src/components/icon/Icon.stories.mdx

```diff
@@ -29,15 +29,15 @@ export const Template = (args) => {
         }}
         argTypes={Utils.hideStyledComponentProperties({
             size: {
+                options: ICON_SIZES,
```

Comment:
is there a reason to extract these from the control object?

Reply:
oh yeah ... in the next storybook version (v7.0.0) it will be deprecated to have it in `control.options` ... this is just to be future-proof